ui/root/src/commonMain/kotlin/social/androiddev/root/root/RootComponentViewModel.kt

```diff
@@ -17,31 +17,23 @@ import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.SupervisorJob
 import kotlinx.coroutines.cancel
 import kotlinx.coroutines.flow.MutableStateFlow
-import kotlinx.coroutines.flow.StateFlow
 import kotlinx.coroutines.flow.asStateFlow
 import kotlinx.coroutines.launch
 import social.androiddev.domain.authentication.model.AuthStatus
 import social.androiddev.domain.authentication.usecase.GetAuthStatus
 import kotlin.coroutines.CoroutineContext
 
-interface RootComponentViewModel : InstanceKeeper.Instance {
-    val authState: StateFlow<AuthStatus?>
-}
-
-fun RootComponentViewModel(coroutineContext: CoroutineContext, getAuthStatus: GetAuthStatus): RootComponentViewModel =
-    RealRootComponentViewModel(coroutineContext, getAuthStatus)
-
-private class RealRootComponentViewModel(coroutineContext: CoroutineContext, getAuthStatus: GetAuthStatus) :
-    RootComponentViewModel {
+class RootComponentViewModel(coroutineContext: CoroutineContext, getAuthStatus: GetAuthStatus) :
```

Comment:
Nit: please format so params are on their own lines for clearer readability/style/formatting

Reply:
Will do - too bad spotless can't do that 😞